### PR TITLE
DATAUP-496: add summary to collapsed cell state

### PR DESCRIPTION
**Pull request by ialarmedalien**
# Description of PR purpose/changes

Adding collapsed cell states for the bulk cell. 
Misc tidying / eslint error fixes in custom.js

# Jira Ticket / Issue #
e.g. https://kbase-jira.atlassian.net/browse/DATAUP-496
- [x] Added the Jira Ticket to the title of the PR (e.g. `DATAUP-69 Adds a PR template`)

# Testing Instructions
* Details for how to test the PR:
- [x] Tests pass locally and in GitHub Actions
- [x] Changes available by spinning up a local narrative and minimising a bulk import that is or has run.

# Dev Checklist:

- [x] My code follows the guidelines at https://sites.google.com/lbl.gov/trussresources/home?authuser=0
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] (JavaScript) I have run Prettier and ESLint on changed code manually or with a git precommit hook

### CODE REVIEW

File: kbase-extension/static/kbase/custom/custom.js [hunk 1]
```diff
@@ -301,9 +301,9 @@ define([
         })();
 
         p.renderMinMax = function () {
-            let $cellNode = $(this.element),
-                metaToggleMode = utils.getCellMeta(this, 'kbase.cellState.toggleMinMax'),
-                toggleMode = $cellNode.data('toggleMinMax');
+            const $cellNode = $(this.element),
```
Comment (ialarmedalien) [inline kbase-extension/static/kbase/custom/custom.js:304]: Fixed all the eslint errors in this file whilst I was looking for some other info.

File: kbase-extension/static/kbase/js/widgets/narrative_core/kbaseCellToolbarMenu.js [hunk 2]
```diff
@@ -254,13 +270,6 @@ define([
                             collapsedCellJobStatus,
                             // options dropdown
                             renderOptions(_cell, events),
-                            // what are these buttons for?
```
Comment (ialarmedalien) [inline kbase-extension/static/kbase/js/widgets/narrative_core/kbaseCellToolbarMenu.js:257]: removed unused buttons

Comment (briehl) [reply kbase-extension/static/kbase/js/widgets/narrative_core/kbaseCellToolbarMenu.js:257]: I don't remember seeing those any time recently, and there's a different triangle button for showing that an app has a new version. I can't tell what would make those not hidden. Maybe those should appear if the app is running?

Comment (briehl) [reply kbase-extension/static/kbase/js/widgets/narrative_core/kbaseCellToolbarMenu.js:257]: But with some manual tinkering and forcing them to show, they look suspiciously out of place. I think they're fine to remove.

File: test/unit/spec/common/props-Spec.js [hunk 3]
```diff
@@ -43,7 +43,7 @@ define(['common/props', 'testUtil'], (Props, TestUtil) => {
             props.setItem('pet', { type: 'dog', name: 'peet' });
             expect(props.getItem('pet')).toEqual({ type: 'dog', name: 'peet' });
         });
-        it('Set two object propertyes', () => {
+        it('Set two object properties', () => {
```
[the remark this answers is not in the document]
Comment (briehl) [reply test/unit/spec/common/props-Spec.js:46]: I kinda liked the 1700s make-up-your-own-spelling style. But, I suppose this is fine.

**Review by briehl — APPROVED** [review]

**Pull request merged**